tests/test_compat.py

```diff
@@ -9,6 +10,34 @@ async def test_create_select_compat_mode(manager):
     obj1 = await manager.create(CompatTestModel, text="Test 1")
     obj2 = await manager.create(CompatTestModel, text="Test 2")
     query = CompatTestModel.select().order_by(CompatTestModel.text)
+    assert isinstance(query, peewee.ModelSelect)
+    result = await manager.execute(query)
+    assert list(result) == [obj1, obj2]
+
+
+@all_dbs
+async def test_compound_select_compat_mode(manager):
+    obj1 = await manager.create(CompatTestModel, text="Test 1")
+    obj2 = await manager.create(CompatTestModel, text="Test 2")
+    query = (
+        CompatTestModel.select().where(CompatTestModel.id == obj1.id) |
+        CompatTestModel.select().where(CompatTestModel.id == obj2.id)
+    )
+    assert isinstance(query, peewee.ModelCompoundSelectQuery)
+    result = await manager.execute(query)
+    assert len(list(result)) == 2
+    assert obj1 in list(result)
+    assert obj2 in list(result)
+
+
+@all_dbs
+async def test_raw_select_compat_mode(manager):
+    obj1 = await manager.create(CompatTestModel, text="Test 1")
+    obj2 = await manager.create(CompatTestModel, text="Test 2")
+    query = CompatTestModel.raw(
+        'SELECT id, text, data FROM compattestmodel m ORDER BY m.text'
+    )
+    assert isinstance(query, peewee.ModelRaw)
```

Comment:
Nice tests but i didn't mention i mean tests for the new api with AioModel. Sorry about that. Could you copy the tests to aio_model/test_selecting and rewrite in new style? We hadn't had tests for such queries at all and i thought it would be good to have a couple ones for each type of query to be sure we don't break anything during development. Compat tests are ok too can be left until we drop the old api

Reply:
@kalombos There is a test for `.raw()` in `test_raw_query()` I'll add a test for compound select.

Reply:
I made search for RawQuery in a hurry so didn't find it. That's good the test exists :) And the test for compound select too :)

Reply:
@kalombos `AioModel` does not have capabilities for `raw()` and compound selects via `aio_execute ()`. Old compatibility API via Manager works, tests are present.

```await TestModel.raw('select id, text from testmodel').aio_execute()```

Produces:

```AttributeError: 'ModelRaw' object has no attribute 'aio_execute'```

Reply:
@rudyryk  Oh, well i have made #237 #236 then